packages/launchpad/src/App.vue

```diff
@@ -4,8 +4,13 @@
   </div>
 
   <div v-else class="bg-white h-full">
-    <HeaderBar :gql="query.data.value.app" />
-    <Wizard :query="query.data.value" />
+    <div v-if="app?.isInGlobalMode">
```

Comment:
Since `App` is non nullable, and we have the check for `query.data.value` in the `v-if`, I think we can just do:

```suggestion
    <div v-if="query.data.value.app.isInGlobalMode">
```

Then we won't need the additional `computed` property, either.